shell/browser/native_window_views_win.cc

```diff
@@ -260,11 +260,12 @@ bool NativeWindowViews::PreHandleMSG(UINT message,
     case WM_SIZING: {
       is_resizing_ = true;
       bool prevent_default = false;
-      NotifyWindowWillResize(gfx::Rect(*reinterpret_cast<RECT*>(l_param)),
-                             &prevent_default);
+      gfx::Rect bounds = gfx::Rect(*reinterpret_cast<RECT*>(l_param));
+      HWND hwnd = GetAcceleratedWidget();
+      gfx::Rect dpi_bounds = ScreenToDIPRect(hwnd, bounds);
+      NotifyWindowWillResize(dpi_bounds, &prevent_default);
       if (prevent_default) {
-        ::GetWindowRect(GetAcceleratedWidget(),
-                        reinterpret_cast<RECT*>(l_param));
+        ::GetWindowRect(hwnd, reinterpret_cast<RECT*>(l_param));
```

Comment:
Anyone know what the purpose of this line is?

Reply:
this line makes very little sense to me 🤔

Reply:
This line is responsible for preventing the window from moving or resizing depending on the event. It's very unclear how it works given the description of [GetWindowRect](https://docs.microsoft.com/en-us/windows/win32/api/winuser/nf-winuser-getwindowrect).

Reply:
Ah, I realized what it's doing. It's updating `l_param` to the current window dimensions prior to being resizing/moved. This is described in the docs for [`WM_SIZING`](https://docs.microsoft.com/en-us/windows/win32/winmsg/wm-sizing).